Parsers/ASimWebSession/Parsers/ASimWebSessionFortinetFortiGate.yaml

```diff
@@ -55,7 +55,7 @@ ParserQuery: |
     | extend 
```

Comment:
please see the errors below:
![image](https://github.com/user-attachments/assets/6bbc3eea-c58b-472f-b486-565e3408e475)

Reply:
please update vim file as well. i.e. Update the parser version and lastupdate property

Reply:
Versions, LastUpdated and References have been fixed.